src/drivers/voxl2_io/voxl2_io.cpp

```diff
@@ -770,11 +774,11 @@ int Voxl2IO::calibrate_escs(){
 
 	/* PWM MAX 3 seconds */
 	PX4_INFO("Writing PWM MAX for 3 seconds!");
-	int16_t max_pwm[4]{VOXL2_IO_MIXER_MAX, VOXL2_IO_MIXER_MAX, VOXL2_IO_MIXER_MAX, VOXL2_IO_MIXER_MAX};
-	if (_debug) PX4_INFO("%i %i %i %i", max_pwm[0], max_pwm[1], max_pwm[2], max_pwm[3]);
-	int16_t led_cmd[4]{0,0,0,0};
-	cmd.len = voxl2_io_create_pwm_packet4_fb(max_pwm[0], max_pwm[1], max_pwm[2], max_pwm[3],
-					       				   led_cmd[0], led_cmd[1], led_cmd[2], led_cmd[3],
+	int16_t max_pwm[8]{VOXL2_IO_MIXER_MAX, VOXL2_IO_MIXER_MAX, VOXL2_IO_MIXER_MAX, VOXL2_IO_MIXER_MAX, VOXL2_IO_MIXER_MAX, VOXL2_IO_MIXER_MAX, VOXL2_IO_MIXER_MAX, VOXL2_IO_MIXER_MAX};
+	// if (_debug) PX4_INFO("%i %i %i %i %i %i %i %i", max_pwm[0], max_pwm[1], max_pwm[2], max_pwm[3], max_pwm[4], max_pwm[5], max_pwm[6], max_pwm[7]);
```

Comment:
This debug statement had to be commented out due to compiler warning for formatting (too many format prints?)